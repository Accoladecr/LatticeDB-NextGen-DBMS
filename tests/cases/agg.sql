CREATE TABLE nums (
    id INT PRIMARY KEY,
    val INT
);
INSERT INTO nums (id, val) VALUES (1,10),(2,20),(3,30);
SELECT MIN(val), MAX(val) FROM nums;
EXIT;
