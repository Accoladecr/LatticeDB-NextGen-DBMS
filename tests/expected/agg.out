CREATE TABLE
INSERT 3 row(s)
min | max
10.000000 | 30.000000
Bye.
